tutorials/vue/search.md

```diff
@@ -0,0 +1,136 @@
+To create a functioning search bar component, do the following:
```

Comment:
Add some more explanation about how the Vue component works. `keyup` triggers `search()`, `search()` makes an HTTP request, display the results.